CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2019-12-12)
+
+- Fixed the modal component to not place content incorrectly behind other neighboring items. [#73](https://github.com/blackbaud/skyux-modals/pull/73)
```

Comment:
LGTM. Just one suggestions (take it or leave it). Should this be written in the postivive tense? Something like...
`Fixed the modal component to correctly stack children elements inside the modal.`?

Reply:
Ooh. I like that

Reply:
I've changed it to your wording.